Parsers/ASimAuthentication/ARM/ASimAuthentication/README.md

```diff
@@ -1,15 +1,18 @@
-# ASIM Authentication Normalization source agnostic parser
+# Microsoft Windows Events ASIM Authentication Normalization Parser
+
+This template deploys the ASIM Authentication schema parser for Microsoft Windows Events.
+
+This Query creates an empty table of the authentication schema.
```

Comment:
Found it here, but needs to be resolved in the YAML. To be consistent with current texts it should be "This function returns an empty ASIM Authentication schema"